ospec.js

```diff
@@ -3,12 +3,16 @@
 if (typeof module !== "undefined") module["exports"] = m()
 else window.o = m()
 })(function init(name) {
-	var spec = {}, subjects = [], results, only = [], ctx = spec, start, stack = 0, nextTickish, hasProcess = typeof process === "object", hasOwn = ({}).hasOwnProperty
+	var spec, subjects = [], results, only = [], ctx, start, stack = 0, nextTickish, hasProcess = typeof process === "object", hasOwn = ({}).hasOwnProperty
 	var ospecFileName = getStackName(ensureStackTrace(new Error), /[\/\\](.*?):\d+:\d+/), timeoutStackName
 	var globalTimeout = noTimeoutRightNow
 	var currentTestError = null
-	if (name != null) spec[name] = ctx = {}
-
+	function reset() {
+		results = null
+		spec = ctx = {}
+		if (name != null) spec[name] = ctx = {}
+	}
+	reset()
```

Comment:
This lets the worker run the suites one file at a time, maximizing the parallelism.